src/core/api/nodopagamenti_api/nodoPerPM/v2/_swagger.json.tpl

```diff
@@ -1,20 +1,15 @@
 {
-  "swagger": "2.0",
+  "openapi": "3.0.1",
```

Comment:
this file can be renamed to `_openapi.json.tpl`, what do you think?

Reply:
Fixed with 07563725